### ISSUE TITLE
typo on LAFF-NLA.pdf page 111

### ISSUE THREAD

**Issue by limbo018** [rvdg/LAFF-NLA#1]
In the last second line,  I think it should be R \in C^{n \times n} instead of R \in C^{m \times n}.

Thanks
Yibo

**Comment by rvdg** [comment]
You are right.  Thank you for reporting.  (It will be fixed the next time I push the file)
